core/node/proof_data_handler/src/tee_request_processor.rs

```diff
@@ -47,49 +51,52 @@ impl TeeRequestProcessor {
     ) -> Result<Option<Json<TeeProofGenerationDataResponse>>, RequestProcessorError> {
         tracing::info!("Received request for proof generation data: {:?}", request);
 
-        let mut min_batch_number = self.config.tee_config.first_tee_processed_batch;
-        let mut missing_range: Option<(L1BatchNumber, L1BatchNumber)> = None;
+        let batch_ignored_timeout = ChronoDuration::days(10);
+        let min_batch_number = self.config.tee_config.first_tee_processed_batch;
 
-        let result = loop {
-            let Some(l1_batch_number) = self
+        loop {
+            let Some(locked_batch) = self
                 .lock_batch_for_proving(request.tee_type, min_batch_number)
                 .await?
             else {
-                // No job available
-                return Ok(None);
+                return Ok(None); // no job available
```

Comment:
nit: this can now be `break`, too... Either change all `break` to `return` or the other way round

Reply:
Fixed [here](https://github.com/matter-labs/zksync-era/pull/2980/commits/5de035be5d8577a07a8685ee7dd1a0ad506d9b99). ✅